src/tests/integration/test_postgreSQL.py

```diff
@@ -7,8 +6,6 @@ def test_meta_cmd_display(ip_with_postgreSQL):
     assert ("public", "taxi", "table", "ploomber_app") in out.result
 
 
-# Known issue, addressing in https://github.com/ploomber/jupysql/issues/90
-@pytest.mark.xfail(reason="known autocommit mode issue")
 def test_auto_commit_mode_on(ip_with_postgreSQL, capsys):
     ip_with_postgreSQL.run_cell("%config SqlMagic.autocommit=True")
     ip_with_postgreSQL.run_cell("%sql CREATE DATABASE new_db")
```

Comment:
run_cell won't raise any exceptions even if the code fails; we must ensure that the property that holds the exc is empty to guarantee that there were no. Also, we should check that the output of the `create database` command prints some confirmation message to ensure that the command executed successfully

Reply:
Fix in [70021af](https://github.com/ploomber/jupysql/pull/91/commits/70021afdb86e2caa3829a5de4e825469b07db206)

Changes: 
1. Check exec has any error
2. the output of the `create database` command doesn't throw any message, we run the manual listing databases command `\l` and assert if new databases gets created here